tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -42,36 +45,203 @@ spec:
     - name: windows-ssh-key-vol
       secret:
         secretName: $(params.windowsSSHKey)
+    - name: mac-ssh-key-vol
+      secret:
+        secretName: $(params.macSSHKey)
   workspaces:
     - name: data
       description: Workspace to save the results to
   results:
+    - name: unsignedMacDigest
+      type: string
+      description: |
+        Digest used by Mac signing host to pull unsignged content via ORAS
+    - name: signedMacDigest
+      type: string
+      description: |
+        Digest used to pull signed Mac content back to pipeline via ORAS
     - name: unsignedWindowsDigest
       type: string
       description: |
-        Digest used by signing host to pull unsignged content via ORAS
+        Digest used by Windows signing host to pull unsignged content via ORAS
     - name: signedWindowsDigest
       type: string
       description: |
-        Digest used to pull signed content back to pipeline via ORAS
+        Digest used to pull signed Windows content back to pipeline via ORAS
   steps:
     - name: push-unsigned-using-oras
       image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
       script: |
-        #!/usr/bin/env bash
+        #!/bin/bash
+        set -x
+
+        CONTENT_DIR=$(workspaces.data.path)/$(params.pipelineRunUid)/$(params.contentDir)
```

Comment:
I would suggest using just `CONTENT_DIR=$(workspaces.data.path)/$(params.contentDir)` and have the parameter point to the full relative path.

But I wonder where this will come from. There will be another task that will fetch the binaries from somewhere? From where? Also, currently the workspace uses a PVC that is 1 GB in size. Will that be enough?

Reply:
`CONTENT_DIR` path updated as such.

> There will be another task that will fetch the binaries from somewhere? 

yes extract-binaries-from-image task is expected to put binaries in that path. 

>  currently the workspace uses a PVC that is 1 GB in size. Will that be enough?

For now it should be enough but when bigger products such as OpenShift migrate to use this workflow we will need more storage. IIRC, each binary of that product is 1GB+ and there will be several binaries here for mac/windows and multiple architectures

Reply:
OK, I guess we can worry about that later. I thought we had an option to provide a PV via data file somewhere, but I can't find it, so maybe not.

I wonder, couldn't the registry be also used for this? The extract-binaries-from-image would push the unsigned binaries to registry via oras. Or the extraction of the binaries could even be part of this task.

Reply:
> I'm not following what that would gain us here?

We wouldn't need to worry about the size of the PV. Because you could use emptyDir which I think is only limited by disk space on the kubernetes node where the pod runs.

Reply:
> CONTENT_DIR path updated as such.

It is correct in the signing task, but now it's incorrect in tasks/publish-to-cgw/publish-to-cgw.yaml:
```
CONTENT_DIR = "$(params.contentDir)"
```
The parameter should always be relative.

Reply:
That was updated in publish-to-cgw.yaml because of how sign-binares returned it. 

I have updated both to match what you have said.

Reply:
Now it's ok, thanks.